packages/ember-glimmer/lib/utils/bindings.js

```diff
@@ -70,34 +70,9 @@ export const AttributeBinding = {
 
     if (attribute === 'style') {
       reference = new StyleBindingReference(reference, referenceForKey(component, 'isVisible'));
-    } else {
-      reference = map(reference, this.mapAttributeValue);
     }
 
     operations.addDynamicAttribute(element, attribute, reference);
-  },
-
-  mapAttributeValue(value) {
```

Comment:
From what I can tell these were undergoing this normalization twice and would cause issues downstream.